packages/paste-core/components/alert/src/index.tsx

```diff
@@ -112,7 +112,7 @@ const Alert = React.forwardRef<HTMLDivElement, AlertProps>(
           <MediaBody as="div">{children}</MediaBody>
           {onDismiss && typeof onDismiss === 'function' && (
             <MediaFigure align="end" spacing="space60">
-              <Button onClick={onDismiss} variant="link" size="reset" element={`${element}_DISMISS_BUTTON`}>
+              <Button onClick={onDismiss} variant="secondary_icon" size="reset" element={`${element}_DISMISS_BUTTON`}>
```

Comment:
Can you update the close icon's title to be "Dismiss alert"? would make the language more concise, the word this doesnt really do anything